js/scenes/Home.js

```diff
@@ -38,6 +38,7 @@ export class Home extends Component {
   componentDidMount() {
     const { dispatch } = this.props;
     dispatch(fetchItemsIfNeeded());
+    dispatch(watchGeoLocation());
```

Comment:
On unmount we need to stop-watching no? Meaning we need to keep count of 'clients' inside `geolocations/index.js` so we know when to actually stop watching.